crates/next-api/src/module_graph.rs

```diff
@@ -512,6 +512,15 @@ async fn validate_pages_css_imports(
     graph.traverse_edges_from_entries(entries, |parent_info, node| {
         let module = node.module;
 
+        // If the module being imported isn't a global css module, there is nothing to validate.
+        let module_is_global_css =
+            ResolvedVc::try_downcast_type::<CssModuleAsset>(module).is_some();
+
+        if !module_is_global_css {
+            return GraphTraversalAction::Continue;
+        }
+
+        // We allow any global import of css files from the `node_modules` folder.
```

Comment:
This line is ambiguous:
- is it valid to import a CSS file that is inside of node_modules: `import "css-reset/style.css";`
- is it valid to import a CSS from a JS file in node_modules

You might be checking the wrong case here

Reply:
That might unintentionally be allowed based on how it's implemented in Webpack, not sure.
---
```suggestion
        // We allow imports of global CSS files which are inside of `node_modules`.
```